src/webcomponents/clinical/clinical-analysis-update.js

```diff
@@ -390,19 +428,23 @@ class ClinicalAnalysisUpdate extends LitElement {
                             field: "status",
                             type: "custom",
                             display: {
-                                render: status => html`
-                                    <clinical-status-filter
-                                        .status="${status.id}"
-                                        .statuses="${this.opencgaSession.study.internal?.configuration?.clinical?.status[this.clinicalAnalysis.type.toUpperCase()]}"
-                                        .multiple=${false}
-                                        .classes="${this.updateParams.status ? "updated" : ""}"
-                                        .disabled="${!!this.clinicalAnalysis?.locked}"
-                                        @filterChange="${e => {
-                                            e.detail.param = "status.id";
-                                            this.onFieldChange(e);
-                                        }}">
-                                    </clinical-status-filter>
-                                `,
+                                render: status => {
+                                    return html`
+                                        <clinical-status-filter
+                                            .status="${status.id}"
+                                            .statuses="${this.opencgaSession.study.internal?.configuration?.clinical?.status[this.clinicalAnalysis.type.toUpperCase()]}"
+                                            .multiple=${false}
+                                            .classes="${this.updateParams.status ? "updated" : ""}"
+                                            .disabled="${!!this._clinicalAnalysis?.locked}"
+                                            @filterChange="${
+                                                e => {
+                                                    e.detail.param = "status.id";
+                                                    this.onFieldChange(e);
+                                                }
+                                            }">
```

Comment:
Please put the function in the same line as the `@filterChange` attribute:
```
@filterChange="${e => {
    e.detail.param = "status.id";
    this.onFieldChange(e);
}}">
```

Reply:
Same as above